frontend/src/service/Api.service.ts

```diff
@@ -0,0 +1,88 @@
+import { AuthService } from './Auth.service';
+import { Err, ErrorService } from './Error.service';
+import { EnvService } from './Env.service';
+import { GraphQLService, IGraphQLError } from './GraphQL.service';
+import { Url } from '../entity/Url';
+
+interface IGraphQLSchemaQuery {
+  authQuery: IGraphQLSchemaAuthQuery;
+}
+
+interface IGraphQLSchemaAuthQuery {
+  URLs?: IGraphQLSchemaURL[];
+}
+
+interface IGraphQLSchemaURL {
+  alias: string;
+  originalURL: string;
+}
+
+const GET_USER_SHORT_LINKS_QUERY = `
```

Comment:
Let's embedded this into the `invokeGetUserShortLinksApi` and rename the function name to `getUserShortLinks`. I don't think this query will be reused in other method of `ApiService`.